src/GRA/Annotations.cs

```diff
@@ -77,6 +78,7 @@ public static class Interface
             public const string AndTheir = "and their {0}";
             public const string Apply = "Apply";
             public const string AvatarSaved = "Avatar saved.";
+            public const string Bundle = "Bundle";
```

Comment:
This annotation is missing from the .resx

Reply:
Resolved in commit b2f1144d79586ad0f605fc9354432a8a201a8f5c